doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -19,10 +22,13 @@ Desktop 2023 R2.
 
 Error installing PyAEDT using pip
 ---------------------------------
-1. **Proxy Server** If your company uses a proxy server you may have to update some settings at the command line.
+1. **Proxy Server** If your company uses a proxy server you may have to update proxy
+   settings at the command line.
    See the `pip documentation <https://pip.pypa.io/en/stable/user_guide/#using-a-proxy-server>`_ for details.
```

Comment:
```suggestion
- **Proxy server**: If your company uses a proxy server, you may have to update proxy
   settings at the command line. For more information, see the `Using a Proxy
   Server<https://pip.pypa.io/en/stable/user_guide/#using-a-proxy-server>`_ in the pip
   documentation.
```
You always want to say why you would want to click a link first and then match the title of the topic where the link takes you and then the source where this link is found.